### ISSUE TITLE
Option for manual planning or setting priorities

### ISSUE THREAD

**Issue by Errors4l** [yznpku/LabCompass#16]
Let me just say that the recently released auto planner is wonderful, making planning a lot faster and easier. The tool also keeps way better track of where you are, and will dynamically alter the route if you take a different exit (or tell you to go back).

The main problem I'm facing now is that you can no longer decide in which order you visit rooms. Although this is no big deal in most cases, it is especially useful when paths branch off. Take the below example of today's layout (January 24th).

![image](https://user-images.githubusercontent.com/2056697/35313480-d7d3ab26-00c0-11e8-8108-0d11fc028572.png)

In this case, the silver door in map 14 can be either at the very beginning or end.  
What I'd normally do in this situation is quickly go from 13 to 14, and check if the door is at the beginning of the map. After that, I'd go back to 13 and proceed to 16. From the trial I'd go back to 14 if the silver door wasn't at the beginning, because it would be at the very end.

Although I don't expect the tool to cover such situations, by allowing us to choose whether we want to use the new or old system, people like me that run into this issue can just switch to manual planning on days where the lab layout presents this kind of problem.

Lab file: 
[uber24jan.json.txt](https://github.com/yznpku/LabCompass/files/1658419/uber24jan.json.txt)
(File has a txt extension because GitHub doesn't seem to support .json)

**Comment by thmour** [comment]
Yes, I've stumbled across this problem many times, including today (2018-04-03). It would be best if we could rearrange the numbers of the plan at the bottom, or add some, and then have a reset button in case we f*** it up, to reset it to the autoplan's default route
